useful/replace.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Please rename this script so its name identifies its purpose in case we have more like this one.